script/Arcueid/Day 13/13_00_ARC13_6.txt

```diff
@@ -78,7 +78,7 @@ His life force is truly something to be admired.
 [sha:16e2f4957ef3acd6c006a48720bd90bd450b89fa]{
 -- Page 452, Offset 32977.
 -- 「“―――なんだ、今のは”」
-"What the hell―――was that?"
+"What the hell――was that?"
```

Comment:
Ehh, dash probably not needed, but at the same time then it might need a rewrite